src/style.css

```
@@ -1 +1,81 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  background: #fff;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  height: 60px;
+  padding: 0;
+  font-size: 12px;
+}
+
+.nav_link:not(:last-child) {
+  margin-right: 20px;
+}
+
+.link {
+  position: relative;
+  padding: 23px 0;
+  font-weight: 500;
+  text-decoration: none;
+  text-align: center;
+  color: #000;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  top: 56px;
+  display: block;
+  content: "";
+  border-radius: 8px;
+  width: 37px;
```

```suggestion
  position: absolute;
  bottom: 0;
  display: block;
  content: "";
  border-radius: 8px;
  width: 100%;
```
in this way it will be more responsive